lib/alexa_generator/interaction_model.rb

```diff
@@ -45,29 +45,39 @@ def initialize(intents, utterance_templates, slot_bindings)
     end
 
     def intent_schema
-      {
-          intents: @intents.values.map do |intent|
-            {
-                intent: intent.name,
-                slots: intent.slots.map do |slot|
-                  {
-                      name: slot.name,
-                      type: slot.type
-                  }
-                end
-            }
-          end
-      }
+      out = { intents: [] }
```

Comment:
we tend to prefer doing it the old fashioned way. been burned by the collects and maps and rejects and such with minor misunderstandings and magic.
Do whatever you want, of course.